### PR TITLE
[HOPS-1340] Fixed the logs for deleted blocks

### PR DESCRIPTION
**Pull request by smkniazi**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added and passed (for bug fixes / features)
- [x] HOPS JIRA issue has been opened for this PR
- [x] All commits have been squashed down to a single commit
- [x] The commit message has the following format: [HOPS-XXX] message

* **Post a link to the associated JIRA issue**
https://hopshadoop.atlassian.net/browse/HOPS-1340

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by berthoug** [comment]
Jenkins test this please.

**Comment by berthoug** [comment]
Jenkins test this please.

**Comment by berthoug** [comment]
Jenkins test this please.

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java [hunk 1]
```diff
@@ -4444,7 +4444,8 @@ public void setUp() throws StorageException {
             LOG.debug("reported block id=" + rdbi.getBlock().getBlockId());
             inodeIdentifier = INodeUtil.resolveINodeFromBlock(rdbi.getBlock());
             if (inodeIdentifier == null) {
-              LOG.error("Invalid State. deleted blk is not recognized. bid=" +
+              if(!rdbi.isDeletedBlock())
+              LOG.warn("Invalid State. deleted blk is not recognized. bid=" +
```
Comment (berthoug) [inline hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java]: can you fix the indentation?

Comment (smkniazi) [reply hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java]: fixed

**Pull request merged**